addons/website_slides_survey/static/src/xml/website_slide_upload.xml

```diff
@@ -18,7 +18,7 @@
                         <div class="img-thumbnail h-100">
                             <div class="o_slide_tutorial p-3">
                                 <div class="h5">How to upload a certification on your course?</div>
-                                <div class="my-4">You can create your certification from here or use an existing one. Once your certification is created, you can still edit it in backend.</div>
+                                <div class="my-4">Select an existing certification from the list. Create or edit a certification can be done from the backend.</div>
```

Comment:
I think "Creating or editing a certification can be done..." would be better, currently sounds a bit off.